src/test/java/lambda/part1/exercise/Exercise3.java

```diff
@@ -0,0 +1,71 @@
+package lambda.part1.exercise;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
+import lambda.data.Person;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.arrayContaining;
+
+@SuppressWarnings({"ConstantConditions", "unused"})
+class Exercise3 {
+
+    @Test
+    void sortPersonsByAgeUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        // TODO use Arrays.sort + expression-lambda
+        Arrays.sort(persons, (p1, p2) -> p1.getAge() - p2.getAge());
```

Comment:
It's safer to use Integer.compare because of the edge values

Reply:
I'm afraid person with negative age can be met hardly ever) Thank you for advice, I have changed implementation.